qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -89,14 +95,14 @@ def solve_fermion(
        of ``40`` or fewer orbitals are supported.
 
     Args:
-        bitstring_matrix: A set of configurations defining the subspace onto which the Hamiltonian
-            will be projected and diagonalized. This is a 2D array of ``bool`` representations of bit
-            values such that each row represents a single bitstring. The spin-up configurations
-            should be specified by column indices in range ``(N, N/2]``, and the spin-down
-            configurations should be specified by column indices in range ``(N/2, 0]``, where ``N``
-            is the number of qubits.
-        hcore: Core Hamiltonian matrix representing single-electron integrals
-        eri: Electronic repulsion integrals representing two-electron integrals
+        hcore: Core Hamiltonian matrix representing single-electron integrals.
+        eri: Electronic repulsion integrals representing two-electron integrals.
+        norb: The number of spatial orbitals.
+        nelec: The numbers of spin up and spin down electrons.
```

Comment:
Can we add a description of ``ci_strings`` here. We should definitely state what happens in default case (HF bitstring used). It may be worth even mentioning that in docstring description. Up to you on that